EstateManagement/Factories/IModelFactory.cs

```diff
@@ -33,7 +40,8 @@ public interface IModelFactory
         /// <param name="merchant">The merchant.</param>
         /// <param name="merchantBalance">The merchant balance.</param>
         /// <returns></returns>
-        MerchantResponse ConvertFrom(Merchant merchant, MerchantBalance merchantBalance = null);
+        MerchantResponse ConvertFrom(Merchant merchant,
+                                     MerchantBalance merchantBalance = null);
```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/manual/stuart_ferguson1/EstateManagement/pullRequest?prid=5907881)